crates/holochain_websocket/examples/nodejs_echo_client/nodejs_echo_client.js

```diff
@@ -29,9 +29,11 @@ class HolochainWebsocket {
   }
```

Comment:
This file actually was supposed to be executable, note the shebang at the top : )

Reply:
Not exactly sure what you mean. This was just copied from the old verison

Reply:
oh, git shows the executable bit changing: `100755 → 100644`

Reply:
zippy has a setup that occasionally *adds* the executable bit to random files like `lib.rs`... there is another file you moved that removed one of these, so I thought you were doing it on purpose : )

Reply:
Oh I see what you mean now